.setup/bin/refresh_vagrant_workers.py

```diff
@@ -0,0 +1,96 @@
+import os
```

Comment:
this script should only be run on vagrant, let's add a check towards the top of the file to make sure this isn't accidentally run on production.  (this script could be damaging to a production install)